Assets/SEE/GraphProviders/SingleGraphPipelineProvider.cs

```diff
@@ -44,13 +46,13 @@ public class PipelineGraphProvider : GraphProvider
         /// <returns></returns>
         /// <remarks>Exceptions may be thrown by each nested graph provider.</remarks>
         public override async UniTask<Graph> ProvideAsync(Graph graph, AbstractSEECity city,
-                                                          Action<float> changePercentage = null,
-                                                          CancellationToken token = default)
+            Action<float> changePercentage = null,
+            CancellationToken token = default)
```

Comment:
Why was this formatting changed? Shouldn't these parameters be indented to the level of the other parameters?